exercises/etl/canonical-data.json

```diff
@@ -0,0 +1,62 @@
+{
+  "transform": {
+    "description": "transforms the input map from a map of value: [ keys ] to a map of key: value for each of the keys",
+    "cases": [
+      {
+        "description": "transforms one value",
+        "input": {
+          "1": ["A"]
+        },
+        "expected": {
+          "a" : 1
+        }
+      },
+      {
+        "description": "tramsforms more values",
```

Comment:
"transforms" is spelled incorrectly (multiple times).

I'd rather see more useful test descriptions.
"Transforms more values" is a possible description for all these tests, what is this test actually testing?